student_class.rb

```diff
@@ -0,0 +1,16 @@
+require './person_class'
+
+class Student < Person
+  def initialize(classroom, name = 'Unknown', parent_permission: true)
+    super(12, name, parent_permission)

```

Comment:
- [ ] This Constructor extends parent's constructor by adding @classroom, like so: 
```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```
Kindly remove the hardcoded value of age.